brainglobe_atlasapi/atlas_generation/atlas_scripts/cavefish_atlas.py

```diff
@@ -0,0 +1,233 @@
+__version__ = "0"
+
+import csv
+import glob as glob
+import os
+import time
+
+# from os import listdir, path
+# p = Path('.')
```

Comment:
Just to simplify things when we come to standardise these scripts, can you delete the commented out code?